health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -0,0 +1,512 @@
+            required: item?.isRequired,
+            isActive: item?.isActive,
+            reGex: item?.isRegex ? item?.regex?.regex : null,
+            order: item?.key,
+            additionalDetails: item // Complete object goes here
+        };
+
+        return questionObject;
+    }
+
+    // Helper function to generate the desired object format for each question
+    function createQuestionObject(item, tenantId, idCodeMap) {
```

Biome

[error] Shouldn't redeclare 'createQuestionObject'. Consider to delete it or rename it.

'createQuestionObject' is defined here:


(lint/suspicious/noRedeclare)